src/components/SingleReview.js

```diff
@@ -0,0 +1,33 @@
+import React from "react";
+import { fetchReviewbyId } from "../api";
+import { useEffect, useState } from "react";
+import { useParams } from "react-router-dom";
+
+const SingleReview = () => {
+  const { review_id } = useParams();
+  const [review, setReviewbyId] = useState({});
+  const [isLoading, setIsLoading] = useState(false);
+  useEffect(() => {
+    setIsLoading(true);
+    fetchReviewbyId(review_id).then((individualReviewFromapi) => {
+      setIsLoading(false);
+      setReviewbyId(individualReviewFromapi);
+    });
+  }, []);

```

Comment:
You'll want to add `review_id` into the dependency array here - react wants to ensure that if it changes, the effect re-runs